phpseclib/Crypt/RSA.php

```diff
@@ -2013,9 +1992,11 @@ function _rsassa_pkcs1_v1_5_sign($m)
     {
         // EMSA-PKCS1-v1_5 encoding
 
+        // If the encoding operation outputs "intended encoded message length too short," output "RSA modulus
+        // too short" and stop.
```

Comment:
This comment does not seem to correspond to the code:
* There is no catching of the "intended encoded message length too short" exception
* There is no output "RSA modulus too short" being made